flyway-commandline/pom.xml

```diff
@@ -55,6 +55,10 @@
             <artifactId>jtds</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.pivotal.jdbc</groupId>
+            <artifactId>greenplumdriver</artifactId>
+        </dependency>
```

Comment:
the greenplumdb is already opensource, https://github.com/greenplum-db
for the driver, I am not sure about it.